### DISCUSSION TITLE
Unable to install successfully following the documentation.

### DISCUSSION THREAD

**Discussion by maimai886** [tamagui/tamagui#1622] [General]
I really like the design style of tamagui, but the official documentation is just too difficult. No matter how I try to install it, I can't get it to work. My project was installed using the "npx react-native@latest init AwesomeProject" method. Is there a complete workflow I can follow?

`npx react-native@latest init AwesomeProject`

`yarn add @tamagui/babel-plugin babel-plugin-transform-inline-environment-variables

yarn addinstall react-native-reanimated
`

Update `babel.config.js`

`yarn add tamagui expo-font @tamagui/font-inter @tamagui/theme-base @tamagui/animations-react-native @tamagui/config react-native-web react-dom
`

Create a `tamagui.config.ts`

I've followed the steps up to this point and still can't successfully use it. I'm not sure what I'm missing.

I hope someone can help me. Thank you.

**Comment by natew** [comment]
We're working on improving this, the Discord is helpful as well.
